### DISCUSSION TITLE
Adding `--stdio` to cmd gives error

### DISCUSSION THREAD

**Discussion by rockorager** [polarmutex/beancount-language-server#79] [Q&A]
Hello! I'm trying to set this up, and when I change my `cmd` from the default to 
```
cmd = { "beancount-langserver", "--stdio"
```
I get the following error:
```
Error executing vim.schedule lua callback: ...l/Cellar/neovim/0.6.0/share/nvim/runtime/lua/vim/lsp.lua:887: RPC[Error] code_name = InternalError, message = "Req
uest initialize failed with message: ENOENT: no such file or directory, open ''"
stack traceback:
        [C]: in function 'assert'
        ...l/Cellar/neovim/0.6.0/share/nvim/runtime/lua/vim/lsp.lua:887: in function 'cb'
        vim.lua:285: in function <vim.lua:285>
```

I've confirmed that beancount-langserver is executable. If I leave the default values in the config and open a beancount file, I can see the server but it isn't attaching (I also have null-ls set up, which IS attaching)

Thoughts?

**Comment by polarmutex** [comment]
let me know if this is still an issue for me to look into

**Reply by rockorager** [reply]
I just tried to set it all back up and I can't even get the language server to show up now (I changed the command to  `beancount-language-server`), so....I'm not sure if the --stdio is still an issue. I can run it with --stdio on the CLI though with no issues.

Sorry, not very helpful
